staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -55,6 +55,121 @@ fn sglist_to_rutabaga_iovecs(
     Ok(rutabaga_iovecs)
 }
 
+pub trait VirtioGpuBackend
+where
+    Self: Sized,
```

Comment:
I don't think the `Self: Sized` trait bound is useful here currently. This would be needed if / when we add a factory method like `fn new(....) -> Self` that returns `Self`. We can add that later, or have another trait for that...